tests/app/publisher/test_publisher.py

```diff
@@ -50,3 +51,17 @@ def test_resolving_message_raises_exception_on_error(publisher):
                 b"test-message",
                 fulfilment_request_transaction_id=str(uuid4()),
             )
+
+
+def test_log_publisher_publish():
```

Comment:
How come we needed this new test?

Reply:
LogPublisher was covered in`test_individual_response.py`